tests/gis_tests/inspectapp/models.py

```diff
@@ -25,3 +25,10 @@ class Fields3D(models.Model):
 
     class Meta:
         required_db_features = ['gis_enabled']
+
+
+class Indexes(models.Model):
+    point = models.PointField(db_index=True)
```

Comment:
I can add test for `point = models.PointField()` but am not sure how to handle the index. Should I pop out the indexes corresponding to spatial fields?